README.md

```diff
@@ -106,6 +106,7 @@ _Web Analytics_
 - [Countly](https://count.ly) - Real time mobile and web analytics, crash reporting and push notifications platform. ([Source Code](https://github.com/countly)) `AGPL-3.0` `Javascript`
 - [Druid](http://druid.io/) - Distributed, column-oriented, real-time analytics data store. ([Source Code](https://github.com/druid-io/druid/)) `Apache-2.0` `Java`
 - [GoAccess](http://goaccess.io/) - Real-time web log analyzer and interactive viewer that runs in a terminal. ([Source Code](https://github.com/allinurl/goaccess)) `GPL-2.0` `C`
+- [Hastic](https://hastic.io) - Hackable time series pattern recognition tool with UI for Grafana ([Source Code](https://github.com/hastic)) `Apache-2.0` `Python` `Nodejs`
```

Comment:
Please can you end the description with a full stop?

Please can you merge the language tags and separate them with a "/" (without a space)?